packages/labs/testing/src/lib/lit-ssr-plugin.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Worker} from 'worker_threads';
+import * as pathlib from 'path';
+import {litSsrPluginCommand} from './constants.js';
+
+import type {TemplateResult} from 'lit';
+import type {TestRunnerPlugin} from '@web/test-runner';
+
+export interface Payload {
+  template: TemplateResult;
+  modules: string[];
+}
+
+export function litSsrPlugin(): TestRunnerPlugin<Payload> {
+  return {
+    name: 'lit-ssr-plugin',
+    async executeCommand({command, payload}) {
```

Comment:
Not blocking for this PR, but would love to think about how this could be factored a little differently so that it could be called from e.g. a koa/express middleware as well, so we're not tied directly to web-test-runner. Should be straightforward, I think?